scrapinghub/client/collections.py

```diff
@@ -91,7 +91,7 @@ def list(self):
 
         :return: a list of collections where each collection is
             represented by a dictionary with ('name','type') fields.
-        :rtype: list[dict]
+        :rtype: list of dicts
```

Comment:
Initial approach led to a lot of Sphinx warnings like `more than one target found for cross-reference u'list'`, so using `list of dicts` was a workaround to overcome this. But Python typing notation (using `List`) works too, let's use it instead 👍